test/round-test.js

```diff
@@ -2,4 +2,152 @@ import chai from 'chai';
 const expect = chai.expect;
 import Puzzle from '../src/puzzle.js'
 import Turn from '../src/turn.js'
-import testData from '../data/test_data_set.js'
\ No newline at end of file
+import testData from '../data/test_data_set.js'
+import Round from '../src/round.js';
+import Player from '../src/player.js';
+import Wheel from '../src/wheel.js'
+
+
+describe('Round', function() {
+  let puzzle1;
+  let testData1;
+  let Player1;
+  let Player2;
+  let Player3;
+  let wheel;
+  let turn;
+  let round;
+  beforeEach(function() {
+
+    testData1 = testData.puzzles.one_word_answers.puzzle_bank[0]
+
+    wheel = new Wheel(testData.wheel);
+    // wheel.selectCurrentValues();
+    Player1 = new Player('Jeffory', 1);
+    Player2 = new Player('Timjax', 2);
+    Player3 = new Player('Karl', 3);
+    puzzle1 = new Puzzle(testData1.category, testData1.number_of_words, 
+      testData1.total_number_of_letters,
+      testData1.first_word, testData1.description, testData1.correct_answer.toUpperCase().split(''))
+    turn = new Turn(puzzle1)
+    round = new Round(wheel, puzzle1, [Player1, Player2, Player3], turn)
+
+  })
+  it('be an instance of round', function() {
+    expect(round).to.be.an.instanceOf(Round)
+  });
+  it('should hold a wheel with six values', function () {
+    wheel.selectCurrentValues();
```

Comment:
Let's add an assertion before `selectCurrentValues` is called that check to see that there is no length. A bit more thorough - just to truly make sure than an update is occurring.